File: PHP/7php 1/cooking_time.php
<?php
/*
|--------------------------------------------------------------------------
| Boiled Eggs
|--------------------------------------------------------------------------
|
*/

function cooking_time(int $eggs): int {
   return ceil($eggs / 8) * 5;
}


function cooking_time($e) {
   return ($e + 7 >> 3) * 5;
}


function cooking_time( int $eggs ) : int {
   for ( $i = $g = 0; $i < $eggs ; $i+=8, $g+=5 ); 
      return $g;
}


function cooking_time(int $eggs, int $n = 0): int {
   return $eggs > 0 ? cooking_time($eggs - 8, $n + 5) : $n;
}


print_r(cooking_time(0) . ' ' . 0 . '
');
print_r(cooking_time(5) . ' ' . 5 . '
');
print_r(cooking_time(10) . ' ' . 10 . '
');

File: Python/7py 1/cooking_time.py
"""
|--------------------------------------------------------------------------
| Boiled Eggs
|--------------------------------------------------------------------------
|
"""

from math import ceil
def cooking_time(eggs):
   return ceil(eggs / 8) * 5


def cooking_time(eggs):
   return (eggs + 7) // 8 * 5


def cooking_time(eggs):
   res = 0
   while( eggs > 0 ):
      res += 5
      eggs -= 8
   return res


def cooking_time(eggs):
   return -(-eggs // 8)*5


def cooking_time(eggs):
   pots, rest = divmod(eggs, 8)
   return (pots + bool(rest)) * 5


print( cooking_time(0), 0)
print( cooking_time(5), 5)
print( cooking_time(10), 10)